scripts/jenkins-support.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+function connect_rally_server_run_test
+{
+    # Connect to rally server and run tests
+    ssh -T root@$rally_server "bash -s $cloud $image_name" <<'EOF'
+cloud=$1
+image_name=$2
+
+# Check if specific cloud deployemnt already exist
+if ! (rally deployment list | grep $cloud); then
```

Comment:
Thinking about it, this will echo whatever matches in the output -- if that behaviour isn't wanted, use grep -q